packages/common/src/utils.ts

```diff
@@ -1,5 +1,81 @@
-import Bignumber from 'bignumber.js'
+import type { Address } from './types.js'
 
-export function weiToEth(wei: string): string {
-  return new Bignumber(wei).div(1e18).toString(10)
+const addressRegex = /^0x[a-fA-F0-9]{40}$/
+
+export const isAddress = (address: string): address is Address => {
+  return addressRegex.test(address)
+}
+
+export const weiHexToEth = (wei: string): string => {
```

Comment:
Does Viem have helper functions already for a lot of these?

Reply:
Yep updated to viem helpers which helps clean this up nicely!